CHANGELOG.md

```diff
@@ -2,86 +2,63 @@
 
 ## Release (1.3.0)
 
-### Ajouts généraux 
-### Commandes ajoutées
+In this release, we decided to use English as the main language.
 
-### Corrections de bogues
+### additions
```

Comment:
Personally, I would remove the section's header (`additions`, which should be `Additions` by the way) if there are no entries for this peculiar release.